src/Molecule.py

```diff
@@ -119,7 +119,7 @@ def draw_molecule(self, drawn_file_name, orient_with_scaffold):
         d.FinishDrawing()
         d.WriteDrawingText(f'{drawn_file_name}.png')
 
-    def drawMoleculeAsByteStream(self, orient_with_scaffold=False):
+    def drawMoleculeAsByteStream(self, orient_with_scaffold=False, size=None):
```

Comment:
I'm not sure why size=None is needed

Reply:
makes size an optional parameter that can but doesnt have to be passed. None is a standard default value if you dont want to define it as a value